beacon-chain/forkchoice/doubly-linked-tree/types.go

```diff
@@ -29,9 +30,11 @@ type Store struct {
 	nodeByRoot                 map[[fieldparams.RootLength]byte]*Node // nodes indexed by roots.
 	nodeByPayload              map[[fieldparams.RootLength]byte]*Node // nodes indexed by payload Hash
 	slashedIndices             map[types.ValidatorIndex]bool          // the list of equivocating validator indices
+	originRoot                 [fieldparams.RootLength]byte           // The genesis Block root
```

Comment:
Is this really just the genesis block root? or it could be check point sync origin root too?

Reply:
Only genesis root, it's used to deal with zero hashes on checkpoints